### DISCUSSION TITLE
适配UE5.3

### DISCUSSION THREAD

**Discussion by BuYi88** [cgerchenhp/UE_TAPython_Plugin_Release#22] [General]
大佬你好，5.3发布了，请问近期有没有编译5.3版本插件的计划啊

**Comment by cgerchenhp** [comment]
有的，预计这个周末更新

**Reply by BuYi88** [reply]
大佬，很抱歉再次打扰，我在尝试你的ChatGPT案例时候，出现了NameError: name 'chameleon_chat' is not defined 问题，是我没配置好 ，还是您发布的v1.1.0 for UE 5.2.1暂时没启用这部分功能呢，如果是没启用的话，5.3版本的会有这个功能吗

**Reply by BuYi88** [reply]
抱歉，我找到问题所在了，是我第三方库路径配置错了，很不好意思叨扰了

**Reply by cgerchenhp** [reply]
没事，欢迎讨论。5.3的来了喽：https://github.com/cgerchenhp/UE_TAPython_Plugin_Release/releases/tag/v1.2.0-ue5.3.0
